sda-commons-web-autoconfigure/src/main/java/org/sdase/commons/spring/boot/web/metadata/MetadataRequestInterceptor.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2022- SDA SE Open Industry Solutions (https://www.sda.se)
+ *
+ * Use of this source code is governed by an MIT-style
+ * license that can be found in the LICENSE file or at
+ * https://opensource.org/licenses/MIT.
+ */
+package org.sdase.commons.spring.boot.web.metadata;
+
+import java.util.Collections;
+import java.util.Enumeration;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Stream;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.web.servlet.HandlerInterceptor;
+
+public class MetadataRequestInterceptor implements HandlerInterceptor {
+
+  private final Set<String> metadataFields;
+
+  public MetadataRequestInterceptor(Set<String> metadataFields) {
+    this.metadataFields = metadataFields;
+  }
+
+  @Override
+  public void afterCompletion(
+      HttpServletRequest request, HttpServletResponse response, Object handler, Exception ex)
+      throws Exception {
+    MetadataContextHolder.clear();
+    HandlerInterceptor.super.afterCompletion(request, response, handler, ex);
```

Comment:
As the method is just an empty default for convenience, we don't need to call super. Or am I missing something?

```suggestion
```